### PR TITLE
PLUTO-705: Upgrade to Selenium 3

### PR DESCRIPTION
**Pull request by vsingleton**

### CODE REVIEW

File: portlet-tck_3.0/pom.xml [hunk 1]
```diff
@@ -216,12 +216,18 @@
...
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
-            <version>2.53.0</version>
-            <scope>test</scope>
-         </dependency>
+         <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-api</artifactId>
+            <version>3.11.0</version>
+            <scope>compile</scope>
```
Comment (ngriffin7a) [inline portlet-tck_3.0/pom.xml:223]: @vsingleton Just curious, why the change from `<scope>test</scope>` to `<scope>compile</scope>`? The only other file in this PR is TCKSimpleTestDriver.java which doesn't seem to benefit from this change?

File: portlet-tck_3.0/pom.xml [hunk 2]
```diff
@@ -216,12 +216,18 @@
...
+         </dependency>
+         <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.11.0</version>
+            <scope>compile</scope>
```
Comment (ngriffin7a) [inline portlet-tck_3.0/pom.xml]: @vsingleton Same question here about the change in scope.

File: portlet-tck_3.0/pom.xml (hunk 1 above)
Comment (vsingleton) [reply portlet-tck_3.0/pom.xml:223]: [ERROR] /Users/apple/Projects/apache.org/asf/portals-pluto/portlet-tck_3.0/driver/src/main/java/javax/portlet/tck/driver/TCKSimpleTestDriver.java:[40,27] package org.openqa.selenium does not exist

File: portlet-tck_3.0/pom.xml (hunk 2 above)
Comment (vsingleton) [reply portlet-tck_3.0/pom.xml]: This one I was able to switch to >test<

**Comment by ngriffin7a** [comment]
See code review commit: https://github.com/apache/portals-pluto/commit/8a0a1d02c04e143ba1dcc9aff0a7400efb5e3478 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ngriffin7a** [comment]
Merged, thank you. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
